src/tm_mad/ceph/postmigrate

```diff
@@ -55,6 +70,8 @@ fi
 
 migrate_other "$@"
 
-exec_and_log "$SSH $SRC_HOST rm -rf $DST_PATH"
+if [ "$SHARED" != "YES" ]; then
```

Comment:
I think that it is much simpler to have something like

```bash
if [ "$SHARED" = "YES" ]; then
    exit 0
fi
```
This way there are only lines added without change of the indention of the other code.